add 2026-08-12.md (#812)
solid/specification: meetings/2026-08-12.md

* If this is your first time, welcome! Please introduce yourself.

---

## Introductions

## Announcements

## Actions Review

* Michal to offer use case for application needs in LDO, regarding app capabilities
* JW: to explore CID WebID interop/migration
  

## Topics

### Demos

### TPAC Session for Solid CG

https://github.com/w3c-cg/solid/issues/84
https://github.com/w3c/tpac2026-meetings/issues/62

* eP: Please keep in mind registration fee, including on-line participations. [Fee waivers available](https://www.w3.org/news-events/tpac/2026/registration/#registration-fee-waiver)
* CB: Who is planning on attending? Do we have boots on the ground to actually make these meetings happen?
* ... To those attending, please recall that you will be participating as members of the Solid CG but [not speaking on behalf of the Solid CG](https://github.com/solid/specification/blob/main/CONTRIBUTING.md#communication). 
* JW: I will be there.
* EB: Maybe I ll be there
* eP: I applied for the fund, maybe I get it.
* ... will contact Theo about Social WG/CG again.

### ~~LLM generated PRs~~

> ### Opening Issues / PRs without taking responsibility for their contents

Reviewer: ```suggestion
### Opening LLM-generated Issues and PRs, without taking responsibility for their contents
```

https://github.com/solid/specification/issues/802
https://github.com/solid/specification/issues/803

* eP: my preference was to discuss the topic first, before potentially closing those PRs
* CB: I thought you had a good month to talk about this topic while I was on vacation, eP.
* _CB updated the title of this topic: from "LLM generated PRs" to "Opening Issues / PRs without taking responsibility for their contents"._
    * https://github.com/jeswr/solid-specs/blob/6f7dbe2ebffd5772af4e21f7ad047a3cf9682cb4/specs/webauthn-reauth/index.html#L116-L120
    * https://github.com/jeswr/solid-specs/blob/6f7dbe2ebffd5772af4e21f7ad047a3cf9682cb4/specs/dpop-sk/index.html#L93-L96
* ... I am sure JW understands that this is just housekeeping in line with W3C guidelines. JW, you are very welcome to simply re-open these issues once you reviewed your proposal and assume responsibility for its contents.
* CB: The proposals should be owned by the one proposing it. When you hand over something you are responsible for it. It is in lines with W3C guideliens.

> * ...: Once you remove disclaimer "I didn't read it myself" you are welcome to re-open it.

Reviewer: ```suggestion
* ... I am sure JW understands that this is just housekeeping in line with W3C guidelines. JW, you are very welcome to simply re-open these issues once you have reviewed your proposals and assume responsibility for their contents.
* CB: The proposals should be owned by the one proposing them. When you hand over something, you are responsible for it. It is in lines with W3C guidelines.
* ...: Once you remove the "I didn't read it myself" disclaimer from each, you are welcome to re-open it.
```

* JW: Its fine for content to be generated or assisted, it just needs to be fully signed-off by the one proposing.
* CB: People use LLMs, its about taking the ownership about what you propose. It is awkward to accept something that hasn't been reviewed.
* JW: There are some issues, I framed it incorrectly when I opened it. I wanted to gage interest for it to be a spec. If there is an interest I will fully sign-off and review.

> * ...: I think there is an interest, do we want WebAuthn if we want FedCM. We are also interested in symmetric key. Query is also of interest.

Reviewer: ```suggestion
* JW: It's fine for content to be generated or assisted, it just needs to be fully signed-off by the one proposing it.
* CB: People use LLMs; it's about taking ownership of what you propose. It is awkward to accept something that hasn't been reviewed.
* JW: There are some issues. I framed them incorrectly when I opened them. I wanted to gauge interest for them to become specs. If there is interest, I will fully sign-off and review.
* ...: I think there is an interest. Do we want WebAuthn if we want FedCM? We are also interested in symmetric keys. Query is also of interest.
```

* eP: It is a new space. We are getting AI into our workflows. So we can see where we land.
* ... myself, I filed issues with CSS, LLM generated, maintainer can close it or not.
* CB: To respond to JW, if you want to reuse the issues, feel free to reopen and clarify.
* ...: on eP, I agree that this is new and we need to gage what is helpful. I wuold encourage everyone to consider that just dumping stuff on people increases others work. We should be mindful of other's people time. 

> * TT: link please, LLM issues tend to be much longer, we should teach them to do tl;dr at the top

Reviewer: ```suggestion
* eP: It is a new space. We are getting AI into our workflows. So we can see where we land.
* ... Myself, I filed LLM-generated issues with CSS; maintainer can close them or not.
* CB: To respond to JW — if you want to reuse the issues, feel free to reopen and clarify.
* ...: On eP, I agree that this is new, and we need to gauge what is helpful. I would encourage everyone to consider that just dumping stuff on people increases their work. We should be mindful of other people's time. 
* TT: Link to W3C LLM guidelines, please? LLM issues tend to be much longer; we should teach them to include `tl;dr` at the top!
```

* CB: https://www.w3.org/TR/llms-standards/
* CB: ... adding AI to the issue, I don't want AI to prompt me
* eP: Would like to comment on the contents of the PRs (without the LLM).
* ... most of the work seems to go to LWS WG
* ... CG effort seems to be low
* ... Does the CG have the capacity to process these proposals?
* ... Maybe IETF would be better venue to get more feedback?
* ... There is a big overlap between those engaged in Solid CG and LWS WG.

> * ... I am wondering whether we should direct proposals to other venues to get constructive feedback.

Reviewer: ```suggestion
* CB: ... adding AI to the issue; I don't want AI to prompt me.
* eP: Would like to comment on the contents of the PRs (without the LLM).
* ... Most of the work seems to go to LWS WG.
* ... CG effort seems to be low.
* ... Does the CG have the capacity to process these proposals?
* ... Maybe IETF would be better venue to get more feedback?
* ... There is a big overlap between those engaged in Solid CG and LWS WG.
* ... I am wondering whether we should direct proposals to other venues to get constructive feedback.
```

* JW: Given DPoP is defined in an IETF RFC, that might be a better venue.
* eP: In LWS, it is up to OAuth best practices whether or not dpop.
* CB: In general it is fine to have those proposals and provide feedback. It is also fine to say it is not the right venue. That said I wouldn't sush away proposal. Member of the CG has this problem and it surfaced here, it is valid to talk with other people first.

> * ...: I don't think that the argument about overlap betwenn Solid CG and LWS WG does the nature of the CG justice. CG is for the creative ideas and incubation. I don't see why the WG has an effect on capactiy of what each one of us can put into the CG.

Reviewer: ```suggestion
* JW: Given DPoP is defined in an IETF RFC, that might be a better venue.
* eP: In LWS, it is up to OAuth best-practices whether or not to DPoP.
* CB: In general, it is fine to have those proposals and provide feedback. It is also fine to say it is not the right venue. That said I wouldn't shush away such proposals. Member of the CG has this problem and it surfaced here; it is valid to talk with other people first.
* ...: I don't think that the argument about overlap between Solid CG and LWS WG does the nature of the CG justice. CG is for the creative ideas and incubation. I don't see why the WG has an effect on capacity of what each one of us can put into the CG.
```

* eP: Agreed, I am not talking about surpressing the proposal. Just noting that there is limited capacity. When it comes to the split between CG and WG, I personally need to choose between CG and WG. So, I don't see priority in the DPoP proposal compared to Authn Suites from the LWS protocol.

> * TT: I sympathise with the need to pick your battles. I participate in ~25 calls across various CGs, WGs, Task Forces, etc. There is different focus between Solid CG and LWS WG. Keeping that mental difference in mind is important. LWS technologies are a piece of what is necessary to achieve the vision of Solid. Solid CG is more about the exploration of the technologies or the pieces of which are necessary to fulfill the vision. Linked Web Storage (LWS) is about where the data gets written and stored and how various applications can work with that data in various locations and what the servers need to provide in functionality to get the data to the client applications (which are not only the clients that we interact with). So, I cannot imagine not being invovled in both the WG and CG. I hope to manage to throw in my bits into the PRs to make sure that the specifications express what the people mean. Food for thought.

Reviewer: ```suggestion
* eP: Agreed, I am not talking about suppressing the proposal. Just noting that there is limited capacity. When it comes to the split between CG and WG, I personally need to choose between CG and WG. So, I don't see priority in the DPoP proposal compared to Authn Suites from the LWS protocol.
* TT: I sympathise with the need to pick your battles. I participate in ~25 weekly calls across various CGs, WGs, Task Forces, etc. There are different foci across Solid CG and LWS WG. Keeping that mental difference in mind is important. LWS technologies are a piece of what is necessary to achieve the vision of Solid. Solid CG is more about the exploration of the technologies, or the pieces of tech, which are necessary to fulfill the vision. Linked Web Storage (LWS) is about where the data gets written and stored, and how various applications can work with that data in various locations, and what the servers need to provide in functionality to get the data to the client applications (which are not only the "clients" that we interact with). So, I cannot personally imagine not being involved in both the WG and CG. I hope to manage to throw enough of my bits into the PRs to make sure that the specifications express what the people mean, such that new readers get the right messages. Food for thought.
```

* eP: I agree in general but I really meant specifically for Solid OIDC, these seems the place where one superseeds the other.
* TT: It is very important that if in the WG there is a MUST, and the other thinks of an alternative that they put as a MAY, then there would be a problem in blocking the alternative path. In this case, the WG's ought to be pushed to drop that MUST back to a SHOULD, such that the CG's MAY remains possible.
* eP: In that light, I would like to invite RZ to bring Virtual Resources to Solid CG. We should make sure that in LWS there is nothing blocking it but we can work on it more here.

> * ... also there is also a W3C Registry Track, where Solid CG might want later to add additional storage capabilities.

Reviewer: ```suggestion
* eP: I agree in general, but I really meant specifically for Solid OIDC, this seems the place where one supercedes the other.
* TT: It is very important that if in the WG there is a MUST, and someone in the CG thinks of an alternative that they would put as a MAY, then there may be a problem in the MUST blocking the alternative (MAY) path. In this case, the WG ought to be pushed to drop that MUST back to a SHOULD, such that the CG's MAY remains possible -- and may eventually become part of the WG's specs!
* eP: In that light, I would like to invite RZ to bring Virtual Resources to Solid CG. We should make sure that there is nothing in LWS blocking it, but we can work on it more here (in CG).
* ... Worth noting, there is also the W3C Registry Track, to which we may later want to add additional storage capabilities.
```

### granting DELETE access on a specific resource

* eP: hit it again during implementation work yesterday, granting DELETE without granting anything else on the parent container
* ... I am using two authorization systems. ACP and custom SAI grants to see if access is allowed or denied.
* ... when only using SAI grants, this is easy.
* ... when using ACP, it seems that you need allow write access
* CB: I'm not aware that this would be the same with WAC. This should be working as you expect and as it works with your sai policy engine.
* ...: ldp:contains are server managed anyways, only if I remove resource the server is reponsible for removing the containment triple, this doesn't requier client/user to have that particiular right.
* eP: I will investigate, including specification-tests repo

> * CB: I will also check WAC, I don't think it should work this way. Possibly CSS issue.

Reviewer: ```suggestion
* eP: I hit it again during implementation work yesterday, granting DELETE without granting anything else on the parent container.
* ... I am using two authorization systems, ACP and custom SAI grants, to see if access is allowed or denied.
* ... When only using SAI grants, this is easy.
* ... When using ACP, it seems that you need allow write access on ???.
* CB: I'm not aware that this would be the same with WAC. This should be working as you expect and as it works with your SAI policy engine.
* ...: `ldp:contains` are server-managed anyway. If I only remove the resource, the server is responsible for removing the containment triple; this doesn't require the client/user to have that particular right.
* eP: I will investigate, including specification-tests repo.
* CB: I will also check WAC. I don't think it should work this way. Possibly CSS issue.
```

### FediPod

* CB: I find it very cool and it worked out of the box for me!
* ...: Great to see interaction between Fediverse and Solid
* ...: app-development channel on matrix
* ... https://github.com/jeff-zucker/FediPod
* eP: he also added AT Protocol
* CB: For me I want to use existing Solid account to access Fediverse 
* ...: Maybe setup script will give you AT Proto user handle, if you go the first round.

> * ...: Fediverse works for sure

Reviewer: ```suggestion
* CB: I find it very cool, and it worked out of the box for me!
* ...: Great to see interaction between Fediverse and Solid.
* ...: `app-development` channel on Matrix.
* ... https://github.com/jeff-zucker/FediPod
* eP: He also added AT Protocol.
* CB: For me, I want to use an existing Solid account to access the Fediverse.
* ...: Maybe setup script will give you an AT Proto user handle, if you go the first round.
* ...: Fediverse works for sure.
```

* CB: The way I understand it, the script gives you two options. You onboard Solid account to the fediverse. Either create fresh solid account, or using existing Solid account.
* eP: does it add webfinger identity for you?
* CB: it adds `foaf:account` triple linking to Fediverse identity
* ...: accounts are seperate but linked

> * eP: does it provide webfinger for you?

Reviewer: ```suggestion
* CB: The way I understand it, the script gives you two options. You onboard a Solid account to the fediverse. Either create a fresh solid account, or use an existing Solid account.
* eP: Does it add a WebFinger identity for you?
* CB: It adds a `foaf:account` triple linking to your Fediverse identity.
* ...: Accounts are separate but linked.
* eP: Does it provide WebFinger for you?
```

* CB: https://1111.solid.aifb.kit.edu/.well-known/webfinger
* ...: I can use mastodon app on my phone and i seee my solid fediverse account

> * AB: There is also aonother one on JSS, I've patched it and my copy is working correctly now and more Solid compliant. I use a real profile card https://github.com/bourgeoa/javascriptsolidserver

Reviewer: ```suggestion
* CB: https://1111.solid.aifb.kit.edu/.well-known/webfinger
* ...: I can use the Mastodon app on my phone, and I see my Solid Fediverse account.
* AB: There is also another one on JSS. I've patched it, and my copy is working correctly now, and is more Solid compliant. I use a real profile card — https://github.com/bourgeoa/javascriptsolidserver
```

### Future CG meeting topic

* AB: Could you add the ODI report on the difficulty of using Solid
* CB: I am afraid I have not seen that
* AB: https://solidproject.solidcommunity.net/2026/Tooling%20research%20-%20summary%20findings.md

> * CB: Will add to next weeks agenda to make people aware of this.

Reviewer: ```suggestion
* AB: Could you add the ODI report on the difficulty of using Solid?
* CB: I am afraid I have not seen that?
* AB: https://solidproject.solidcommunity.net/2026/Tooling%20research%20-%20summary%20findings.md -> https://solidproject.solidcommunity.net/2026/Tooling%20research%20-%20summary%20findings.pdf
* CB: Will add to next weeks agenda to make people aware of this.
```
